index.html

```diff
@@ -22,14 +22,14 @@
     <div class="container nav-container">
       <h1 class="logo-heading">Fun Bus</h1>
       <nav class="nav">
-        <a class="nav-link" href="#">Home</a>
-        <a class="nav-link" href="#">About Us</a>
-        <a class="nav-link" href="#">Blog</a>
-        <a class="nav-link" href="#">Contact</a>
+        <a class="nav-link1" href="#">Home</a>
+        <a class="nav-link2" href="#">About Us</a>
+        <a class="nav-link3" href="#">Blog</a>
+        <a class="nav-link4" href="#">Contact</a>
```

Comment:
Look into datasets and how to implement them to keep the same class name throughout your css. You could also `forEach` through them to do something to them individually.